nilber-calculator/src/Components/Comma.js

```diff
@@ -0,0 +1,27 @@
+// This function assumes what is being submitted to it is a string, with a decimal point and two places after the decimal. 
+
+const CommaFormatted = (number) => {
+	var delimiter = ","; // replace comma if desired
+	var a = number.split('.',2)
+	var d = a[1];
+	var i = parseInt(a[0]);
+	if(isNaN(i)) { return ''; }
+	var minus = '';
+	if(i < 0) { minus = '-'; }
+	i = Math.abs(i);
+	var n = new String(i);
+	var a = [];
+	while(n.length > 3) {
+		var nn = n.substr(n.length-3);
+		a.unshift(nn);
+		n = n.substr(0,n.length-3);
+	}
+	if(n.length > 0) { a.unshift(n); }
+	n = a.join(delimiter);
+	if(d.length < 1) { amount = n; }
+	else { amount = n + '.' + d; }
+	amount = minus + amount;
+	return amount;
+}
+
+//Code Developed by Chris Coyier
```

Comment:
🤔 Maybe you shouldn't include someone else's code in your app. It's nice that you give credit though. The code isn't working/being used in your app anyway.
For the comma you can slice off the decimal part and use the `toLocaleString` method on the rest. Like this `yourLargeNumber.toLocaleString()` - this should do the trick. Then add whatever you sliced off before.

Reply:
I will work on adding the toLocaleString method.